### PR TITLE
New release to add the enhancement requested in issue #19

### PR DESCRIPTION
**Pull request by genegr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,12 +117,13 @@ Authentication is used by the exporter as the mechanism to cross authenticate to
...
-| http://\<exporter-host\>:\<port\>/metrics/array   | endpoint       | Array metrics        |
-| http://\<exporter-host\>:\<port\>/metrics/clients | endpoint       | Clients metrics      |
-| http://\<exporter-host\>:\<port\>/metrics/usage   | endpoint       | Quotas usage metrics |
+| URL                                                | GET parameters | description               |
+| ---------------------------------------------------| -------------- | --------------------------|
+| http://\<exporter-host\>:\<port\>/metrics          | endpoint       | Full array metrics        |
+| http://\<exporter-host\>:\<port\>/metrics/array    | endpoint       | Array metrics             |
+| http://\<exporter-host\>:\<port\>/metrics/clients  | endpoint       | Clients metrics           |
+| http://\<exporter-host\>:\<port\>/metrics/usage    | endpoint       | Quotas usage metrics      |
+| http://\<exporter-host\>:\<port\>/metrics/polucies | endpoint       | NFS policies info metrics |
```
Comment (sdodsley) [inline README.md:126]: typo: policies

File: README.md [hunk 2]
```diff
@@ -64,6 +64,47 @@ The provided dockerfile can be used to generate a docker image of the exporter.
...
 docker build -t pure-fb-ome:$VERSION .
 ```
+### Authentication
+
+Authentication is used by the exporter as the mechanism to cross authenticate to the scraped appliance, therefore for each array it is required to provide the REST API token for an account that has a 'readonly' role. The api-token can be provided in two ways
+
+- using the HTTP Authorization header of type 'Bearer', or
+- via a configuration map in a specific configuration file.
+
+The first option requires to specify the api-token value as the authorization parameter of the specific job in the Prometheus configuration file.
```
Comment (sdodsley) [inline README.md:74]: change `to specify` to `specifying`

File: README.md [hunk 3]
```diff
@@ -64,6 +64,47 @@ The provided dockerfile can be used to generate a docker image of the exporter.
...
+The second option provides the FlashBlade/api-token key-pair map for a list of arrays in a simple YAML configuration file that is passed as paramether to the exporter. This makes possible to write more concise Prometheus configuration files and also to configure other scrapers that cannot use the HTTP authentication header.
```
Comment (sdodsley) [inline README.md:75]: typo: parameter

**Review by sdodsley — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (genegr) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (genegr) [reply README.md]: Fixed

File: README.md (hunk 1 above)
Comment (genegr) [reply README.md]: Fixed

**Review by sdodsley — APPROVED** [review]

**Pull request merged**